app/src/main/java/com/keylesspalace/tusky/MainActivity.java

```diff
@@ -415,7 +417,10 @@ public void cancel(ImageView imageView) {
                             Intent intent = new Intent(MainActivity.this, EditProfileActivity.class);
```

Comment:
Maybe not in this PR but this if-else chain *screams* "make me switch statement"

Reply:
yes true!